packages/popover/src/popover.tsx

```diff
@@ -76,11 +76,21 @@ if (__DEV__) {
   PopoverAnchor.displayName = "PopoverAnchor"
 }
 
+export interface PopoverTriggerProps {
+  /**
+   * The React child to use as the
+   * trigger for the popover
+   */
+  children: React.ReactChild
+}
+
 /**
  * PopoverTrigger opens the popover's content. It must be an interactive element
  * such as `button` or `a`.
  */
-export const PopoverTrigger: React.FC = (props) => {
+export const PopoverTrigger: React.FC<
+  React.PropsWithChildren<PopoverTriggerProps>
+> = (props) => {
```

Comment:
I think we can simplify this props type definition to

```suggestion
export const PopoverTrigger = (props: PopoverTriggerProps) => {
```

Reply:
Thanks, @TimKolberger. Much simpler. Resolved.